symbiflow-xray-yosys/meta.yaml

```diff
@@ -1,7 +1,7 @@
 {% set version = '%s_%04i_%s'|format(GIT_DESCRIBE_TAG|replace('v','') or '0.X', GIT_DESCRIBE_NUMBER|int, GIT_DESCRIBE_HASH or 'gUNKNOWN') %}
 
 package:
-  name: yosys
+  name: symbiflow-xray-yosys
```

Comment:
I don't think the SymbiFlow is X-Ray specific?
Probably should just be `symbiflow-yosys`?

Reply:
Changed to `symbiflow-yosys`